libs/react-components/community/src/components/form/file-upload/file-upload.stories.tsx

```diff
@@ -85,11 +88,28 @@ export const PdfAndTxtOnly: Story = {
   },
 };
 
+const MaxSizeTemplate: StoryFn<ComponentProps<typeof FileUpload>> = (args) => {
+  const [invalidSizeFiles, setInvalidSizeFiles] = useState<File[]>([]);
+
+  return (
+    <VerticalSpacing>
+      <FileUpload {...args} onInvalidSize={setInvalidSizeFiles} />
+      {invalidSizeFiles.length > 0 && (
+        <Notification type="error">
```

Comment:
Nice-to-have, but i think more "nice" and semantically correct is showing that error with fileUpload.helper error/important text.
Also when uploading new files, we should reset invalidSizeFiles. Because that error is not relevant anymore

Reply:
Switched to formhelper and reset on change.